src/internal/cli/commands/NetworksCommand.ts

```diff
@@ -16,13 +24,32 @@ export const NetworksCommand = new commander.Command("networks")
 
     filteredNetworks.sort((a, b) => a.sort_order - b.sort_order);
 
-    filteredNetworks.forEach((network) => {
-      table.push([network.ethereum_network_id ,network.slug]);
-    });
+    table.push(
+      ...(await Promise.all(
+        filteredNetworks.map(async (network) => {
+          if (verbose) {
+            const blockNumber = await TenderlyService.getLatestBlockNumber(
+              network.ethereum_network_id
+            );
+
+            return [network.ethereum_network_id, network.slug, blockNumber];
+          } else {
+            return [network.ethereum_network_id, network.slug];
+          }
+        })
```

Comment:
Like it 😎

Reply:
Props to @zdenkolini for help with that 😎

Reply:
thanks for the praise 🤗